views/help/people.hbs

```diff
@@ -32,7 +32,7 @@
 						</div>
 						<div id="collapse-{{@index}}" class="collapse show in">
 							<div class="card-block">
-								Das Hasso-Plattner-Institut steht für alle Fragen technischer Natur bereit. Bei Problemen mit der Schul-Cloud selbst oder bei Schwierigkeiten bezüglich der Bedienung verwende bitte das Kontaktformular oder sende eine E-Mail an:<br><a href="mailto:feedback@schul-cloud.org" target="blank" rel="noopener" class="externalLink">feedback@schul-cloud.org</a>
+								{{$t "help.text.theHassoPlattnerInstituteIsAvailable" }}<br><a href="mailto:feedback@schul-cloud.org" target="blank" rel="noopener" class="externalLink">feedback@schul-cloud.org</a>
```

Comment:
fetch the email adresses from configuration, at least `feedback@schul-cloud.org`